centreon/www/include/monitoring/external_cmd/popup/massive_downtime.php

```diff
@@ -38,14 +38,16 @@
     exit();
 }
 
+const DOWNTIME_ON_HOST = 75;
+
 $select = array();
 if (isset($_GET['select'])) {
     foreach ($_GET['select'] as $key => $value) {
-        if ($cmd == '75') {
-            $tmp = preg_split("/\;/", $key);
+        if ((int) $cmd == DOWNTIME_ON_HOST) {
+            $tmp = preg_split("/\;/", urlencode($key));
```

Comment:
[urlencode](https://www.php.net/manual/en/function.urlencode.php) ? not [rawurlencode](https://www.php.net/manual/en/function.rawurlencode.php) (RFC 3986) ?